job-server-extras/test/spark.jobserver/python/PythonSparkContextFactorySpec.scala

```diff
@@ -159,7 +162,7 @@ class PythonSparkContextFactorySpec extends FunSpec with Matchers with BeforeAnd
       val jobData = jobDataOrProblem.get
       val result = job.runJob(context, jobEnv, jobData)
       result should matchPattern {
-        case m: java.util.Map[_, _] if m.asScala.toMap == Map("a" -> 2, "b" -> 3, "c" -> 1) =>
+        case m: java.util.Map[_, _] if JMapWrapper(m).toMap == Map("a" -> 2, "b" -> 3, "c" -> 1) =>
```

Comment:
ok, changed back. I just thought import scala.collection.JavaConverters._ is not allowed by scala-style.